my.css

```diff
@@ -1,6 +1,10 @@
+html {
+  font-size: 16px; 
+}
+
```

Comment:
this is the base font-size for the `rem` introduced. One big problem: This pins the entire website to 16px and creates a major accessibility problem. This will override any customized font from the browser which is usually use as the base (and usually matches 16px anyway). You should use percentage values here and relate on that common standard of having 16px as the base font size of the browser.

People can typically still resize with zoom text in/out but the browser base font-size is disabled with that so it's not the best to introduce.